packages/help-center/src/components/help-center-contact-form.tsx

```diff
@@ -393,6 +400,34 @@ export const HelpCenterContactForm = () => {
 		}
 	};
 
+	const getSupportedLanguages = ( supportType: string, locale: string ) => {
+		const isLiveChatLanguageSupported = (
+			config( 'livechat_support_locales' ) as Array< string >
+		 ).includes( locale );
+
+		const isLanguageSupported = ( config( 'upwork_support_locales' ) as Array< string > ).includes(
+			locale
+		);
+
+		switch ( supportType ) {
+			case SUPPORT_HAPPYCHAT:
+				return ! isLiveChatLanguageSupported;
+
+			case SUPPORT_TICKET:
```

Comment:
Would it be possible to not have duplicate code and make the FAB use an extracted version of this function, that we can put inside the Help Center?